spec/controllers/admin/roles_controller_spec.rb

```diff
@@ -36,19 +36,35 @@
         create(:role, name: 'I')
       end
 
-      it 'orders the roles list by column and direction DESC' do
+      it 'orders the recordings list by column and direction DESC' do
```

Comment:
? why this change

Reply:
It's roles not recordings xD
I copied the test from recordings spec but forgot to alter the test description.

Reply:
Don't know probably mistyped during a rebase.
Will change it.